test/syscalls/linux/kill.cc

```diff
@@ -102,6 +102,62 @@ TEST(KillTest, CanKillAllPIDs) {
   EXPECT_EQ(0, WEXITSTATUS(status));
 }
 
+// reaper process of pid namespace will ignore blockable
+// signal with SIG_DFL action, such as SIGTERM
+TEST(KillTest, CannotTermReaper) {
+  pid_t pid = 1;
+  int status;
+  EXPECT_THAT(kill(pid, SIGTERM), SyscallSucceeds());
+  ASSERT_THAT(RetryEINTR(waitpid)(pid, &status, 0),
```

Comment:
Wouldn't we get ECHILD here even if the signal was delivered and pid 1 exited?  (Assuming our test was able to continue running....)

Reply:
Thanks, here just want to check wether the reaper process still alive, I will changed into send signal 0 to the reaper process.

Actually, without currently patch, this check code won't reached, cause send SIGTERM to reaper will cause the test container exit, then got a case failed.